### ISSUE TITLE
Trakt.tv 405 Error

### ISSUE THREAD

**Issue by mikeyounge** [zakkarry/retraktarr#7]
I’m testing out the script for the first time and was able to successfully authenticate with Trakt on the first run. The config file was updated and shows the two tokens. However, when I run the script, I’m getting a 405 error after the validation of the config file. Not sure what I'm doing wrong.

```
~/.local/bin/retraktarr --oauth REDACTED

retraktarr v1.0.4
Validating Configuration File: /home/yms/.config/retraktarr.conf

Authorization Code:  REDACTED
Access Token:  REDACTED
Refresh Token:  REDACTED
Your configuration file was successfully updated with your access/refresh token.
```

```
CONFIG FILE ~/.config/retraktarr.conf
[Trakt]
client_id = REDACTED
client_secret = REDACTED
username = REDACTED
redirect_uri = https://google.com
oauth2_token = REDACTED
oauth2_refresh = REDACTED

[Radarr]
url = https://myradarrinstance
api_key = REDACTED
trakt_list = TestListMovies
trakt_list_privacy = private

[Sonarr]
url = https://mysonarrinstance
api_key = REDACTED
trakt_list = TestListShows
trakt_list_privacy = private
```

```
~/.local/bin/retraktarr -r --tag films -c

retraktarr v1.0.4
Validating Configuration File: /home/yms/.config/retraktarr.conf

Trakt.tv Error: Unexpected status code return: 405.
```
If there is a place where the debug logs are stored, I will happily look at those as well and pass them along. Thanks for any help you can provide.

**Comment by zakkarry** [comment]
On lunch break right now but will be home later and if you have discord you can find me in servarr, cross seed, or sonarr discord but there arent currently any debug logs, its been on my todo for a while but ive been busy with crossseed.

will double check everything obvious when im home regarding replicating if i can

my discord user is @zak.ary btw

**Comment by zakkarry** [comment]
It's possible I found what could be causing your issue when attempting to replicate a `-c` run....and I think I see the problem (or just another problem, which would be fortunate I suppose for me, not necessarily you...)

As I previously said, it would be great to have you test my fix before I publish an update. I'll be on Discord this evening.

FYI: 405 is method not found, meaning its possible that the command was malformed and this potentially could have caused it, although I received a stack trace instead of an unexpected status code. Requires confirmation whether this is linked.

**Comment by zakkarry** [comment]
Turns out this was due to punctuation being present in the username for Trakt, specifically, but perhaps not limited to for anyone else, a period in this case.

@mikeyounge changed his username to remove the period and the script functioned properly, the bug I discovered was merely a coincidence.

I will investigate further whether this is something on retraktarr's end or Trakt itself and update this issue.
